Realm/Realm/Realm.cs

```diff
@@ -675,6 +675,44 @@ public void Write(Action action)
             transaction.Commit();
```

Comment:
Can we implement this `Write` in terms of the new one? I know it's a fairly trivial method, but the less code we have, the less bugs we'll ship 😄